examples/deepseek/conf/config_deepseek_v3.yaml

```diff
@@ -10,7 +10,7 @@ experiment:
     backend: vllm
     entrypoint: null
   runner:
-    hostfile: examples/deepseek/conf/hostfile.txt
```

Comment:
I think it's appropriate to allow users to provide relative host file paths in the current directory, which we can then convert to absolute paths.